ocs_samples/basic_samples/SDS/DotNet/SdsClientLibraries/SdsClientLibraries/Program.cs

```diff
@@ -58,7 +56,6 @@ public static async Task<bool> MainAsync(bool test = false)
 
             // LoggerCallbackHandler.UseDefaultLogging = false;
```

Comment:
I just saw this... We shouldn't have any code commented out, can you please remove it. Are we even Logging anything here? I'm so confused now how is this here.

Reply:
No, I don't know why that is there.  I will remove.